Directory.Packages.props

```diff
@@ -15,7 +15,8 @@
     <PackageVersion Include="Amazon.Lambda.SQSEvents" Version="2.2.0" />
     <PackageVersion Include="AWSSDK.Core" Version="4.0.0.2" />
     <PackageVersion Include="AWSSDK.SQS" Version="4.0.0.1" />
-    <PackageVersion Include="AWSSDK.S3" Version="4.0.0.1"/>
+    <PackageVersion Include="AWSSDK.S3" Version="4.0.0.1" />
+    <PackageVersion Include="Supernova.Enum.Generators" Version="1.0.17" />
```

Comment:
This `ItemGroup` is for `AWS` packages.

Could we remove `NetEscapades.EnumGenerators` all together in favor of `Supernova.Enum.Generators` or should we make that a follow up PR?

Reply:
https://github.com/elastic/docs-builder/pull/1226/commits/732899de0d9ca39165b3679a73bcdf8684eeae23

I'd rather do it in a follow-up. Which I can do right after this is merged.